regression-tests/test-results/clang-12-c++20/pure2-last-use.cpp.output

```diff
@@ -1,12 +1,21 @@
-pure2-last-use.cpp2:273:36: error: expected variable name or 'this' in lambda capture list
```

Comment:
Thanks! Regressions weren't re-run so I did that and committed the results.

Note these Clang-12 results in particular changed, can you check the diffs and see if they're benign?

Reply:
Yes, the change looks right.